### ISSUE TITLE
npm i yields: "appears to  be git repo or submodule"

### ISSUE THREAD

**Issue by ccondrup** [halloverden/norwegian-data-validators-ts#3]
Steps to reproduce:
1. Install this package in a project.
My package.json reads:
````json
"dependencies": {
    "@halloverden/norwegian-data-validators-ts": "^2.4.1",
}
````

2. Add _another_ package to project, run npm i to install
Example:
Add to package.json dependencies: `"string-width": "^4.1.0"`
Run `npm install`

3. npm halts with the following message:

````bash
⋊> ~/p/m/docker on feature/user-info ↑ npm i
11:50:50
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.9
npm WARN optional Skipping failed optional dependency /@angular/compiler-cli/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.9
npm ERR! Linux 4.15.0-50-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "i"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! path /home/user/projects/myproj/node_modules/@halloverden/norwegian-data-validators-ts
npm ERR! code EISGIT

npm ERR! git Appears to be a git repo or submodule.
npm ERR! git     /home/user/projects/myproj/node_modules/@halloverden/norwegian-data-validators-ts
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/user/projects/myproj/docker/npm-debug.log
````

Attached log-file: [npm-debug.log](https://github.com/halloverden/norwegian-data-validators-ts/files/3231857/npm-debug.log)

**Comment by ccondrup** [comment]
> npm checks all directories for the existance of a .git directory, and throws the EISGIT error if it exists

Ref: 
1. https://stackoverflow.com/questions/27356915/how-to-update-npm-modules-ignoring-a-git-repo
2. https://github.com/shama/napa/issues/53

**Comment by erichjsonfosse** [comment]
Should be fixed with d6affa5 (released in 2.4.4)
